draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -4433,7 +4434,7 @@ transmission.</t></li>
 the destination addresses it can send to (not each source-destination pair but
 for each destination).
 The parameters SHOULD decay if the address is not used for a long enough time
-period. <xref target='RFC5681'/> specifies this long enough time as a
+period. <xref target='RFC5681'/> specifies this "long enough time" as a
```

Comment:
What about using:
```
<xref target='RFC5681'/> specifies this period of time as a retransmission
timeout.</t></li>
```

Reply:
The next revision will contain the text change suggested by me above. If you don't agree, please unresolve the conversation.